packages/atomic/cypress/utils/network.ts

```diff
@@ -52,3 +54,16 @@ export function getAnalyticsAt(selector: string, order: number) {
   }
   return getAnalytics(selector);
 }
+
+export function modifySearchResultAt(
```

Comment:
so we mock the response with fake data this time?

Reply:
it's more solid, I'm just modifying the first result with a custom one so I know how the templates should respond, will come in handy for other template tests